Python/optimizer_bytecodes.c

```diff
@@ -546,7 +546,9 @@ dummy_func(void) {
         PyFunctionObject *func = (PyFunctionObject *)(this_instr + 2)->operand;
         DPRINTF(3, "func: %p ", func);
         if (func == NULL) {
-            goto error;
+            DPRINTF(3, "\n");
+            DPRINTF(1, "Missing function\n");
+            goto done;
```

Comment:
FWIW, the [function version cache changes](https://github.com/python/cpython/pull/117028) I am planning would solve this better.

Reply:
This isn't about the missing function, but how we handle imperfect optimization.